src/clj/rems/workflow/dynamic.clj

```diff
@@ -174,6 +187,11 @@
   (is (= (set (keys event-schemas))
          (set (get-event-types)))))
 
+(defmethod apply-event [:application.event/created :workflow/dynamic]
+  [application _workflow event]
+  ;; TODO
```

Comment:
What is to do here?

Reply:
Populating the application from the event instead of relying the legacy application model. The second step in #852 will take care of that.

Trying to do that is outside the scope of this PR, because it would require modifying existing code and create too much risk of a merge conflict.

Reply:
TODO comment should say so then?